quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/widget/rest/SourceDataResourceAssembler.java

```diff
@@ -0,0 +1,30 @@
+package com.github.reflectoring.infiniboard.quartermaster.widget.rest;
+
+import java.util.List;
+
+import org.springframework.hateoas.mvc.ResourceAssemblerSupport;
+
+import com.github.reflectoring.infiniboard.packrat.source.SourceData;
+
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.linkTo;
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.methodOn;
+
+public class SourceDataResourceAssembler extends ResourceAssemblerSupport<List<SourceData>, SourceDataResource> {
+
+    private Integer dashboardId;
+    private String  widgetId;
+
+    public SourceDataResourceAssembler(Integer dashboardId, String widgetId) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)